var/spack/repos/builtin/packages/py-ipykernel/package.py

```diff
@@ -25,7 +25,8 @@ class PyIpykernel(PythonPackage):
     version('4.1.0', '638a43e4f8a15872f749090c3f0827b6')
 
     depends_on('python@2.7:2.8,3.3:')
-    depends_on('py-setuptools', type='build')
+    depends_on('python@3.3:', when='@5:')
```

Comment:
```suggestion
    depends_on('python@3.4:', when='@5:', type=('build', 'run'))
```

Reply:
that should then also go for the original one above right?